tensorboard/plugins/mesh/test_utils.py

```diff
@@ -94,4 +94,4 @@ def deserialize_array_buffer_response(byte_content, data_type):
     Flat numpy array with the data.
   """
   return np.frombuffer(byte_content, dtype=data_type)
-  
\ No newline at end of file
+
```

Comment:
The proper fix here is to delete the empty line; please either do that
or just revert this change, because the file is otherwise unrelated.